privacy-security-explainer.md

```diff
@@ -0,0 +1,68 @@
+
+# Protection types
+WebXR must be structured to ensure end users are protected from developers gathering and using sensitive information inappropriately. The necessary protections will vary based on the sensitive data being guarded, and, in some cases, more than one protection is necessary to adequately address the potential threats exposed by specific sensitive information.
+
+## Documents and origins
+When sensitive information can be exposed, the requesting document must be:
+* [Responsible](https://html.spec.whatwg.org/multipage/webappapis.html#responsible-document)
+* Of a [secure context](https://w3c.github.io/webappsec-secure-contexts/#secure-contexts)
+* The [currently focused area](https://html.spec.whatwg.org/multipage/interaction.html#currently-focused-area-of-a-top-level-browsing-context)
+* Of the [same origin-domain](https://html.spec.whatwg.org/multipage/origin.html#same-origin-domain) as the [active document](https://html.spec.whatwg.org/multipage/browsers.html#active-document)
+* Of an origin not blocked by [feature policy](#feature-policy)
+* **TODO** Address the outcome of [#732](https://github.com/immersive-web/webxr/issues/732)
+
+### Focus and visibility
+**TODO** Fill this in with what is agreed upon in [#724](https://github.com/immersive-web/webxr/issues/724) and [#696](https://github.com/immersive-web/webxr/issues/696).
+
+### Feature policy
+**TODO** Fill this in with what is agreed upon in [#308](https://github.com/immersive-web/webxr/issues/308), [#729](https://github.com/immersive-web/webxr/issues/729), [#730](https://github.com/immersive-web/webxr/issues/730), and [#731](https://github.com/immersive-web/webxr/issues/731).
+
+## Trusted UI
+**TODO** Fill this in with what is agreed upon in [#718](https://github.com/immersive-web/webxr/issues/718) and [#719](https://github.com/immersive-web/webxr/issues/719).
+
+## User intention
+It is often necessary to be sure of user intent before exposing sensitive information or allowing actions with a significant effect on the user's experience. This intent may be communicated or observed in a number of ways.
+
+### User activation
+[User activation](https://html.spec.whatwg.org/multipage/interaction.html#activation) is defined within the HTML spec as an action the user can take which can result in certain types of HTML elements becoming activated. For example, a button becomes activated when a user clicks it with a mouse. The concept of user activation, differentiates user-caused events from injected events to prevent pages from spoofing user actions. User activation can be checked by the UA prior to session creation as well as during a session in response to appropriate `XRInputSource` events.
```

Comment:
> User activation can be checked by the UA prior to session creation as well as during a session in response to appropriate `XRInputSource` events.

More accurate to say something like this?

> User activation is required by the UA prior to `immersive` session creation, and is produced during a session in response to appropriate `XRInputSource` events.

Reply:
I was actually trying to avoid being specific about how these protections will be applied to WebXR concretely since that's what I'll cover in the Protected Functionality section.  Perhaps this instead:
> Within a WebXR session, `XRInputSource.select`is also considered to be triggered by user activation.  For more information, see [Input events](input-explainer.md#input-events).

Reply:
That phrasing works for me!